src/EditorFeatures/Core/Implementation/IntelliSense/QuickInfo/IntellisenseQuickInfoBuilder.cs

```diff
@@ -47,6 +55,18 @@ internal static IntellisenseQuickInfoItem BuildItem(ITrackingSpan trackingSpan,
                 quickInfoItem.Sections.Where(s => s.Kind != QuickInfoSectionKinds.Description)
                                       .Select(BuildClassifiedTextElement));
 
+            // build text for RelatedSpan
+            if (quickInfoItem.RelatedSpans.Any())
+            {
+                IEnumerable<ClassifiedSpan> classifiedSpans = null;
+                foreach (var span in quickInfoItem.RelatedSpans)
+                {
+                    classifiedSpans = await Classifier.GetClassifiedSpansAsync(document, span, cancellationToken).ConfigureAwait(false);
```

Comment:
so you only care last span? shouldn't it do classifiedSpans.Concat? or classifiedSpans.AddRange or something?